web/src/features/charts/elements/AreaGraph.tsx

```diff
@@ -211,6 +211,13 @@ function AreaGraph({
     [setGraphIndex, setSelectedLayerIndex]
   );
 
+  const [headerHeight, setHeaderHeight] = useState<number>(0);
+
+  useEffect(() => {
+    const height = document.querySelectorAll('header')[0].offsetHeight;
+    setHeaderHeight(height * 1.1);
+  }, [window.innerWidth, window.innerHeight]);
+
```

Comment:
This seems to be identical to the code in the other file, would be good to extract it to a function so it can be reused instead of duplicated.